src/components/product/second-hand.json

```diff
@@ -0,0 +1,19 @@
+{
+  "collectionName": "components_product_second_hands",
+  "info": {
+    "displayName": "Second Hand"
+  },
+  "options": {},
+  "attributes": {
+    "canBeUsed": {
+      "type": "boolean",
+      "default": false,
+      "required": true
+    },
+    "priceAdjustment": {
+      "type": "integer",
```

Comment:
thought: currently this has no default, which probably makes sense, though I am wondering if we start using this value in a calculation and it defaults to 0 will it make the calculation of the value 0? We could get around this with a falsy check when the actual calculation is done, though that could be an issue if we ever want a value adjustment to actually be 0. I am not sure if that is a scenario that would realistically ever happen.

Reply:
Looks good.